public/pages/MonitorDetails/containers/AnomalyHistory/AnomalyHistory.js

```diff
@@ -0,0 +1,140 @@
+import ContentPanel from '../../../../components/ContentPanel';
+import { AnomaliesChart } from '../../../CreateMonitor/components/AnomalyDetectors/AnomaliesChart';
+import { FeatureChart } from '../../../CreateMonitor/components/AnomalyDetectors/FeatureChart/FeatureChart';
+import DateRangePicker from '../MonitorHistory/DateRangePicker';
+
+const DEFAULT_ANOMALY_TIME_WINDOW_DAYS = 5;
+class AnomalyHistory extends Component {
+  constructor(props) {
+    super(props);
+    this.initialStartTime = moment(Date.now())
+      .subtract(DEFAULT_ANOMALY_TIME_WINDOW_DAYS, 'days')
+      .startOf('day');
+    this.initialEndTime = moment(Date.now());
+    this.state = {
+      startTime: this.initialStartTime,
+      endTime: this.initialEndTime,
+    };
+  }
+  handleRangeChange = (startTime, endTime) => {
+    this.setState({ startTime, endTime });
+  };
+  render() {
+    const { detectorId, monitorLastEnabledTime } = this.props;
+    return (
+      <ContentPanel
+        title="Anomaly results"
+        titleSize="s"
+        bodyStyles={{ minHeight: 200 }}
+        actions={[
+          <DateRangePicker
+            initialStartTime={this.initialStartTime}
+            initialEndTime={this.initialEndTime}
+            onRangeChange={this.handleRangeChange}
+          />,
+        ]}
+      >
+        <AnomalyDetectorData
+          preview={true}
+          startTime={this.state.startTime.valueOf()}
+          endTime={this.state.endTime.valueOf()}
+          detectorId={detectorId}
+          render={anomalyData => {
+            let featureData = [];
+            //Skip disabled features showing from Alerting.
+            featureData = get(anomalyData, 'detector.featureAttributes', [])
+              .filter(feature => feature.featureEnabled)
+              .map((feature, index) => ({
+                featureName: feature.featureName,
+                data: anomalyData.anomalyResult.featureData[feature.featureId] || [],
+              }));
+            const annotations = get(anomalyData, 'anomalyResult.anomalies', [])
+              .filter(
+                anomaly => anomaly.anomalyGrade > 0 && anomaly.startTime >= monitorLastEnabledTime
+              )
+              .map(anomaly => ({
+                coordinates: {
+                  x0: anomaly.startTime,
+                  x1: anomaly.endTime,
+                },
+                details: `There is an anomaly with confidence ${anomaly.confidence}`,
+              }));
+            if (
+              !anomalyData.isLoading &&
+              get(anomalyData, 'anomalyResult.anomalies', []).length === 0
+            ) {
+              return (
+                <EuiEmptyPrompt
+                  style={{ maxWidth: '45em' }}
+                  body={<EuiText>No anomalies found for this detector.</EuiText>}
+                />
+              );
+            }
+            return true ? (
```

Comment:
true? if always true, we can directly return following charts.

Reply:
Good catch. will remove the testing code.